### PR TITLE
6.0/ui/fix switchablegroup value

### PR DESCRIPTION
**Pull request by nhaagen**
Follow-up to #2225; rendering of Switchable Group used to throw (in examples, e.g.)
The issue is with $value beeing set or not rather than subgroups.

### CODE REVIEW

**Comment by nhaagen** [comment]
Added a test for this not to happen again. @klees, @dkloepfer : please have a look.

File: src/UI/Implementation/Component/Input/Field/Renderer.php [hunk 1]
```diff
@@ -592,6 +592,7 @@ protected function renderSwitchableGroupField(Component\Input\Field\SwitchableGr
 		}
 		$options_html = $input_tpl->get();
 
+
```
Comment (klees) [inline src/UI/Implementation/Component/Input/Field/Renderer.php]:
```suggestion
```

File: src/UI/Implementation/Component/Input/Field/SwitchableGroup.php [hunk 2]
```diff
@@ -87,6 +86,9 @@ public function withValue($value) {
 	 */
 	public function getValue() {
 		$key = Input::getValue();
+		if(is_null($key)){
+			return $key;
```
Comment (klees) [inline src/UI/Implementation/Component/Input/Field/SwitchableGroup.php]:
```suggestion
			return null;
```
The value is supposed to be `null` here, `$key` just acidentially is `null` here as well.

**Review by klees — CHANGES_REQUESTED** [review]
Hi @nhaagen, thx a lot. LGTM in general, just minor changes...

**Pull request merged**